movement/plots/occupancy.py

```diff
@@ -0,0 +1,185 @@
+        figure and axes are created and returned.
+    kwargs : Any
+        Keyword arguments passed to ``matplotlib.pyplot.hist2d``
+
+    Returns
+    -------
+    matplotlib.pyplot.Figure
+        Plot handle containing the rendered 2D histogram. If ``ax`` is
+        supplied, this will be the figure that ``ax`` belongs to.
+    matplotlib.axes.Axes
+        Axes on which the histogram was drawn. If ``ax`` was supplied,
+        the input will be directly modified and returned in this value.
+    dict[str, numpy.ndarray]
+        Information about the created histogram (see Notes).
+
+    Notes
+    -----
+    In instances where the counts or information about the histogram bins is
+    desired, the ``return_hist_info`` argument should be provided as ``True``.
+    This will force the function to return a second output value, which is a
+    dictionary containing the bin edges and bin counts that were used to create
+    the histogram.
+
+    For data with ``Nx`` bins in the 1st spatial dimension, and ``Ny`` bins in
+    the 2nd spatial dimension, the dictionary output has key-value pairs;
+    - ``xedges``, an ``(Nx+1,)`` ``numpy`` array specifying the bin edges in
+    the 1st spatial dimension.
+    - ``yedges``, an ``(Ny+1,)`` ``numpy`` array specifying the bin edges in
+    the 2nd spatial dimension.
+    - ``counts``, an ``(Nx, Ny)`` ``numpy`` array with the count for each bin.
+
+    ``counts[x, y]`` is the number of datapoints in the
+    ``(xedges[x], xedges[x+1]), (yedges[y], yedges[y+1])`` bin. These values
+    are those returned from ``matplotlib.pyplot.Axes.hist2d``.
+
+    Examples
+    --------
+    Simple use-case is to plot a histogram of the centroid of all
+    keypoints, aggregated over all individuals.
+
+    >>> from movement import sample_data
+    >>> from movement.plots import plot_occupancy
+    >>> positions = sample_data.fetch_dataset(
+    ...     "SLEAP_three-mice_Aeon_proofread.analysis.h5"
+    ... ).position
+    >>> plot_occupancy(positions)
+
+    However, one can restrict the histogram to only counting the positions of
+    (the centroid of) certain keypoints and/or individuals.
+
+    >>> print("Available individuals:", positions["individuals"])
+    >>> plot_occupancy(
+    ...     positions,
+    ...     # use only one keypoint in computation of centroid
+    ...     keypoints="centroid",
+    ...     # only aggregate for two individuals
+    ...     individuals=[
+    ...         "AEON3B_TP1",
+    ...         "AEON3B_TP2",
+    ...     ],
+    ... )
+
+    ``kwargs`` are passed to the ``matplotlib`` backend as keyword-arguments to
+    this function.
+
+    >>> plot_occupancy(
+    ...     positions,
+    ...     # use only one keypoint in computation of centroid
+    ...     keypoints="centroid",
+    ...     # only aggregate for two individuals
+    ...     individuals=[
+    ...         "AEON3B_TP1",
+    ...         "AEON3B_TP2",
+    ...     ],
```

Comment:
If you switch to using the dataset I proposed:
```suggestion
    >>> plot_occupancy(  
    ...     positions,  
    ...     # plot the centroid of keypoints located on the head  
    ...     keypoints=["snout", "leftear", "rightear"],  
    ...     # only plot data for 1 individual  
    ...     individuals="individual1",  
```